### DISCUSSION TITLE
Welcher Sensor ist der aktuelle Hausverbrauch?

### DISCUSSION THREAD

**Discussion by benzoesaeure** [weltenwort/home-assistant-rct-power-integration#321] [Q&A]
Hallo zusammen,
ich habe 2 RCT Wechselrichter und den RCT Power switch. In meiner App sehe ich immer schön den aktuellen Hausverbrauch ( In der App unten links das Häuschen) leider finde ich genau diesen Wert nicht unter den über 100 Entitäten. Kann mir jemand einen Tipp geben wie ich diesen finde?
 
Hello everyone,
I have 2 RCT inverters and the RCT power switch. In my app I always see the current home consumption (in the app at the bottom left the little house) unfortunately I can't find exactly this value among the more than 100 entities. Can someone give me a tip how to find this?

**Comment by weltenwort** [comment]
Hi @benzoesaeure,

sorry for making it hard to find. The entity for the current household power usage would be `sensor.rct_ps_10_1_consumer_power`. The integrated household energy is represented by the `sensor.rct_ps_10_1_consumer_energy_consumption_{day,month,year,total}` family of entities.

I tried to document this in the README chapter [Most commonly used entities > Household consumers and producers]( https://github.com/weltenwort/home-assistant-rct-power-integration#household-consumers-and-producers). If you have any ideas of how I could make the docs more useful, please let me know.

Hope this helps?
